test/core/ticktree/TickTreeCleanOffer.t.sol

```diff
@@ -6,42 +6,42 @@ import {TickTreeTest, TestTickTree} from "./TickTreeTest.t.sol";
 import "mgv_src/MgvLib.sol";
 import "mgv_lib/Debug.sol";
 
-// Tests of Mangrove.cleanByImpersonation's interaction with the tick tree.
+// Tests of Mangrove.cleanByImpersonation's interaction with the bin tree.
 //
 // The tests use the following pattern:
-// 1. we establish a Mangrove tick tree where there may be offers at:
-//   - the offer to be cleaned's tick (including the offer itself)
+// 1. we establish a Mangrove bin tree where there may be offers at:
+//   - the offer to be cleaned's bin (including the offer itself)
 //   - a higher tick
 //   - a lower tick
-// 2. we take a snapshot of Mangrove's tick tree
-// 3. we clean the offer in both Mangrove and in the snapshot tick tree
-// 4. we check that Mangrove's tick tree matches the test tick tree.
+// 2. we take a snapshot of Mangrove's bin tree
+// 3. we clean the offer in both Mangrove and in the snapshot bin tree
+// 4. we check that Mangrove's bin tree matches the test bin tree.
 //
 // The scenarios we want to test are:
 // - offer fails/doesn't fail
 // - cleaning tick
-//   - tick is a *tick of interest* (ToI) as listed in TickTreeTest
+//   - bin is a *bin of interest* (ToI) as listed in TickTreeTest
```

Comment:
ToI -> BoI

Reply:
fixed in 5ff12ac03bae78f540e71b471e3e655475c46eb7